### PR TITLE
feature/xsiam-brocade-switch-modeling-enhancement-ciac-7337

### PR DESCRIPTION
**Pull request by cweltPA**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-7337

## Description
This PR contains - 

- Modeling rules enhancement for adding additional event and fields mappings. 
- Parsing rules minor issue fix. 
- README.md update for including syslog configuration documentation on Brocade switch appliances. 

## Must have
- [x] Tests
- [x] Documentation 

**No mandatory dependencies were added on this pull request.**

### CODE REVIEW

File: Packs/BrocadeSwitch/README.md [hunk 1]
```diff
@@ -1,17 +1,52 @@
...
+<~XSIAM>
+
 This pack includes XSIAM content.
 
-## Collect Events from Vendor
+## Configuration on Server Side
+This section describes the configuration that needs to be done on Brocade Fabric OS switch appliances in order to forward their event logs to Cortex XSIAM Broker VM via syslog. 
+
+### Syslog Forwarding Configuration
+Brocade Fabric OS switches supports forwarding the audited events to a remote syslog server. The syslog forwarding configuration is done via the [***syslogAdmin***](https://techdocs.broadcom.com/us/en/fibre-channel-networking/fabric-os/fabric-os-commands/9-2-x/Fabric-OS-Commands/syslogAdmin.html) command.
```
Comment (ShirleyDenkberg) [inline Packs/BrocadeSwitch/README.md:10]:
```suggestion
Brocade Fabric OS switches support forwarding the audited events to a remote syslog server. The syslog forwarding configuration is done via the [***syslogAdmin***](https://techdocs.broadcom.com/us/en/fibre-channel-networking/fabric-os/fabric-os-commands/9-2-x/Fabric-OS-Commands/syslogAdmin.html) command. 
```

File: Packs/BrocadeSwitch/README.md [hunk 2]
```diff
@@ -1,17 +1,52 @@
...
 You can configure the specific vendor and product for this instance.
-1. Navigate to **Settings** -> **Configuration** -> **Data Broker** -> **Broker VMs**. 
-2. Right-click, and select **Syslog Collector** -> **Configure**.
-3. When configuring the Syslog Collector, set:
-   - vendor as brocade
-   - product as switch
\ No newline at end of file
+
+1. Navigate to **Settings** &rarr; **Configuration** &rarr; **Data Broker** &rarr; **Broker VMs**. 
+2. Go to the apps tab and add the **Syslog** app for the relevant broker instance. If the **Syslog** app already exists, hover over it and then click **Configure**.
```
Comment (ShirleyDenkberg) [inline Packs/BrocadeSwitch/README.md:42]:
```suggestion
2. Go to the apps tab and add the **Syslog** app for the relevant broker instance. If the **Syslog** app already exists, hover over it and click **Configure**.
```

File: Packs/BrocadeSwitch/README.md [hunk 3]
```diff
@@ -1,17 +1,52 @@
...
+3. Click **Add New**.
+3. When configuring the Syslog Collector, set the following parameters:
+   | Parameter     | Value    
+   | :---          | :---                    
+   | `Protocol`    | Select **UDP** for the default forwarding, or **Secure TCP** if the syslog forwarding on the Brocade switch appliance was defined with the **-secure** option (see [SyslogAdmin](https://techdocs.broadcom.com/us/en/fibre-channel-networking/fabric-os/fabric-os-commands/9-2-x/Fabric-OS-Commands/syslogAdmin.html) command reference).  
+   | `Port`        | Enter the syslog service port that Cortex XSIAM Broker VM should listen on for receiving forwarded events from Brocade Fabric OS switch appliances (This should be aligned with the *-port* operand used on the switch appliance when running the [SyslogAdmin](https://techdocs.broadcom.com/us/en/fibre-channel-networking/fabric-os/fabric-os-commands/9-2-x/Fabric-OS-Commands/syslogAdmin.html) command as described in the [Syslog Forwarding Configuration](#syslog-forwarding-configuration) section).
```
Comment (ShirleyDenkberg) [inline Packs/BrocadeSwitch/README.md:48]:
```suggestion
   | `Port`        | Enter the syslog service port that Cortex XSIAM Broker VM should listen on for receiving forwarded events from Brocade Fabric OS switch appliances. (This should be aligned with the *-port* operand used on the switch appliance when running the [SyslogAdmin](https://techdocs.broadcom.com/us/en/fibre-channel-networking/fabric-os/fabric-os-commands/9-2-x/Fabric-OS-Commands/syslogAdmin.html) command as described in the [Syslog Forwarding Configuration](#syslog-forwarding-configuration) section).   
```

**Comment by ShirleyDenkberg** [comment]
@evisochek Doc review completed.

**Review by evisochek — APPROVED** [review]

**Pull request merged**